_rules/auto-update-text-efbfc7.md

```diff
@@ -77,7 +73,7 @@ The [instruments][instrument] used to pass this rule (if any), must meet all lev
 
 #### Passed Example 1
 
-This `span` element contains text content that is automatically **changed** multiple times without user intervention and there is a button available to stop the automatic changes. The rule is not applicable to the second `p` element because it has a **child changed** (the `span` element).
+This `span` element contains text content that automatically changes multiple times without user intervention and there is a button available to stop the automatic changes. The rule is not applicable to the second `p` element because its child [text node][] is unchanged.
```

Comment:
This is where we need to be clear about what "contains text content" means.
The `span` has **text nodes** that change (are replaced by other text nodes). But the actual [data](https://dom.spec.whatwg.org/#concept-cd-data) of the individual text nodes doesn't change (as far as I remember).

I guess we'll end up saying that "contains text content" refers to the `innerText` 🤔